### PR TITLE
Added support to give name for store dev tool

### PR DESCRIPTION
**Pull request by roopkt**
This is the changes to put name on store dev tool configuration object for the  issue #463

### CODE REVIEW

**Comment by roopkt** [comment]
Finished passing name to devtools fixes #463

**Comment by roopkt** [comment]
Hi @brandonroberts  I updated this pull request with my latest changes.

**Comment by roopkt** [comment]
Hi @brandonroberts ,
I fixed code merge issue as well. Please check if the implementation or the fix for the issue #463 is correct.

**Comment by brandonroberts** [comment]
@roopkt the changelog shouldn't be included in your changes. Please verify that your changes change the name in the extension also.

**Comment by roopkt** [comment]
Hi @brandonroberts  thanks for giving me the direction to pass options to send method. I am planning to inject then ```STORE_DEVTOOLS_CONFIG``` into ```DevtoolsExtension``` so that we can always pass the entire option in send method. This is one potential solution that I think of.

I even wanted to validate by running example app however unfortunately when I run 
``` yarn run eample:star``` I always get error ```NONEMPTY dir``` related error. I tried deleting '../platform/dist' folder manually still when I run the start script I get error at some point while "Copying type definition files" step. I event tried opening new command terminal to execute still same issue. Have you encountered this issue is there any wise workaround of this.

**Comment by roopkt** [comment]
Hey @brandonroberts  
I pushed one more change-set this time I verified the ReduxDevToolExtension name is indeed changing. 👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,204 @@
-# Changelogs
```
Comment (brandonroberts) [inline CHANGELOG.md]: This file should not be included in the changes

File: modules/store-devtools/src/instrument.ts [hunk 2]
```diff
@@ -64,12 +64,16 @@ export function noMonitor(): null {
   return null;
 }
 
+export const DEFAULT_NAME = 'ngrx-store-devtool-instance';
```
Comment (brandonroberts) [inline modules/store-devtools/src/instrument.ts]: Change to `NgRx Store DevTools`

File: modules/store-devtools/src/instrument.ts [hunk 3]
```diff
@@ -64,12 +64,16 @@ export function noMonitor(): null {
...
+
 export function createConfig(
   _options: StoreDevtoolsOptions
 ): StoreDevtoolsConfig {
-  const DEFAULT_OPTIONS: StoreDevtoolsConfig = {
+  const DEFAULT_OPTIONS: Partial<StoreDevtoolsConfig> = {
```
Comment (brandonroberts) [inline modules/store-devtools/src/instrument.ts]: Why is this a partial?

File: CHANGELOG.md (hunk 1 above)
Comment (brandonroberts) [inline CHANGELOG.md]: The changelog should not be included in these changes. We'll generate a changelog in the next release

Comment (roopkt) [reply CHANGELOG.md]: sure

Comment (roopkt) [reply CHANGELOG.md]: Thanks for explaination

File: modules/store-devtools/src/instrument.ts (hunk 2 above)
Comment (roopkt) [reply modules/store-devtools/src/instrument.ts]: I changed the name to NgRx Store DevTools

File: modules/store-devtools/src/instrument.ts (hunk 3 above)
Comment (roopkt) [reply modules/store-devtools/src/instrument.ts]: I removed the partial from here.

**Comment by brandonroberts** [comment]
Thanks!

**Pull request merged**